### ISSUE TITLE
Impossible to use App Secret in a browser

### ISSUE THREAD

**Issue by amakhrov** [GetStream/stream-js#516]
When I try to use api secret in my local environment, it throws `You are publicly sharing your App Secret. Do not expose the App Secret in browsers`.

The README has the following note on the subject:
> If you are running backend code in Google Cloud **or you know what you're doing**, you can specify browser: false in options to skip this check.

Well, I know what I'm doing (it's going to be a developer tool only available locally, with api secret read from environment).
Yet, the suggestion `specify browser: false` doesn't work. It doesn't just suppress the error. It also tells the stream-js client to use Node packages (`http.Agent`) - resulting in a run-time error in a browser. https://github.com/GetStream/stream-js/blob/main/src/client.ts#L257 .

As such, the note in README is misleading. It feels like a mistake in the code rather then the docs though.

Thanks

**Comment by ferhatelmas** [comment]
This repo is about feeds. You are referring another repo: https://github.com/getstream/stream-chat-js which is chat client.
Relevant line is here: https://github.com/GetStream/stream-chat-js/blob/01bf0acd0ac7b13ca9b0e3e61a3f45820fadb1a2/src/client.ts#L164

**Comment by amakhrov** [comment]
Nope, it's about feeds, specifically in relation with token generation:
https://github.com/GetStream/stream-js#api-client-setup-node--browser

**Comment by ferhatelmas** [comment]
I see your point now, will make a patch.

**Comment by ferhatelmas** [comment]
[8.0.2](https://www.npmjs.com/package/getstream/v/8.0.2) should fix your use-case, let me know how it goes.

**Comment by amakhrov** [comment]
Thanks, looks good!
